sklearn/metrics/pairwise.py

```diff
@@ -105,12 +105,12 @@ def check_pairwise_arrays(X, Y, precomputed=False, dtype=None):
 
     if Y is X or Y is None:
         X = Y = check_array(X, accept_sparse='csr', dtype=dtype,
-                             estimator=estimator)
+                            estimator=estimator)
     else:
         X = check_array(X, accept_sparse='csr', dtype=dtype,
-                         estimator=estimator)
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Reply:
apologies, apparently pycharm automatically made those unwanted changes after a faulty click. This has been reverted now and only specific changes remain.